src/chromeDebug.ts

```diff
@@ -14,9 +16,10 @@ const EXTENSION_NAME = 'debugger-for-chrome';
 declare let VERSION: string;
 let versionWithDefault = typeof VERSION === 'undefined' ? 'unspecified' : VERSION; // Not built with webpack for tests
 
-// non-.txt file types can't be uploaded to github
-// also note that __dirname here is out/
-const logFilePath = path.resolve(__dirname, '../vscode-chrome-debug.txt');
+// non-.txt file types can't be uploaded to github. Generated filename is something like: vscode-chrome-debug_2017-07-19_161210.txt
+const timestampForFilename = new Date().toISOString().replace(/T/, '_').replace(/:/g, '').replace(/\..*/, '');
```

Comment:
I only added the timestamp to avoid losing the logs the second time we run the debugger. Would appending to the log file rather than starting from scratch work for you?